woocommerce/wc-admin
# Remove the use of preloaded countries

Fixes #8373 

Removes the use of `countries` from our preloaded settings and moved it to a data store. We only use countries in a few places ( Store address in onboarding, advanced filter in customer analytics, shipping task ). It's a pretty large dataset to be loading on all WooCommerce Admin pages, so this removes that.

### Screenshots

### Detailed test instructions:

- Go to the Onboarding wizard `/wp-admin/admin.php?page=wc-admin&path=%2Fsetup-wizard` and make sure the country/region dropdown still works as expected
- Go to **WooCommerce > Customers** and click the **All Customers** dropdown and select **Advanced Filters**
- Select the country region advanced filter and add some countries and click filter, make sure everything works as expected.
- Go to a unfinished 'Set up shipping' task
- Make sure the shipping task works as expected and the country label is displayed correctly.
